apps/omg_watcher/lib/application.ex

```diff
@@ -38,12 +38,12 @@ defmodule OMG.Watcher.Application do
         OMG.API.EthereumEventListener,
         [
           %{
-            synced_height_update_key: :last_depositer_block_height,
+            synced_height_update_key: :last_depositer_eth_height,
             service_name: :depositer,
             block_finality_margin: block_finality_margin,
             get_events_callback: &OMG.Eth.RootChain.get_deposits/2,
             process_events_callback: &OMG.API.State.deposit/1,
-            get_last_synced_height_callback: &OMG.Eth.RootChain.get_root_deployment_height/0
+            get_last_synced_height_callback: &OMG.DB.last_depositer_eth_height/0
```

Comment:
To not get sync issues, we need to remember to initialize that to root_deployment height, or take maximum of root deployment height and last depositer eth height.